x-pack/filebeat/Makefile

```diff
@@ -1,8 +1,3 @@
-BEAT_NAME=filebeat
-ES_BEATS?=../..
-XPACK_BEAT_PATH?=github.com/elastic/beats/x-pack/${BEAT_NAME}
-GOPACKAGES?=$(shell go list ${BEAT_PATH}/... ${XPACK_BEAT_PATH}/... | grep -v /vendor/ | grep -v /scripts/cmd/ )
-LICENSE=Elastic
```

Comment:
It's not needed here because all of the real build logic originates from the mage in this case so we only need to set `mage.BeatLicense = "Elastic"` in the magefile.go.